server/forms.py

```diff
@@ -558,6 +558,17 @@ class ExportAssignment(BaseForm):
     anonymize = BooleanField('Anonymize', default=False,
                              description="Enable to remove identifying information from submissions")
 
+
+class AssignSlipCalculatorForm(BaseForm):
+    timescale = SelectField('Time Scale', default="days",
+                            choices=[(c.lower(), c.title()) for c in TIMESCALES],
+                            description="Time scale for slip calculation.")
+    show_results = BooleanField('Show Results', default=False)
```

Comment:
Not sure if this is useful as an option. We should either always show the results, or don't